BR-InterviewQuestions.md

StarboardHook opened the pull request:
updated Questions for the interview questions

caraz22 committed:
Cara's questions

caraz22 committed:
hopefully fixed spacing

AvaMcintosh committed:
Update BR-InterviewQuestions.md

hmcintos committed:
Update BR-InterviewQuestions.md

Hunters Questions

hmcintos committed:
Update BR-InterviewQuestions.md

tried to fix spacing issues

StarboardHook committed:
UPDATED FILE: BR-InterviewQuestions.md

StarboardHook committed:
Update BR-InterviewQuestions.md

AvaMcintosh committed:
Update Agile-MVP-BusinessNeedsAnalysis.md

caraz22 committed:
fixed spacing maybe?

caraz22 committed:
more spacing

caraz22 committed:
fixed formatting

caraz22 committed:
more fixing

caraz22 committed:
hopefully a fix

StarboardHook committed:
Update Agile-MVP-CoreFeatures.md

StarboardHook committed:
Update Agile-MVP-CoreFeatures.md

StarboardHook committed:
Update Agile-MVP-CoreFeatures.md

StarboardHook committed:
Update Agile-MVP-CoreFeatures.md

AvaMcintosh committed:
Update Agile-MVP-CoreFeatures.md

AvaMcintosh committed:
Update Agile-MVP-CoreFeatures.md

StarboardHook committed:
Update Agile-MVP-CoreFeatures.md

AvaMcintosh committed:
Update Agile-MVP-CoreFeatures.md

pattonsgirl commented:
Hi guys!  You are doing great work, but this Pull Request is pointing to the "golden" repo ([cs3900-f24-cs3900-CS3900-GP-Starter](https://github.com/WSU-kduncan/cs3900-f24-cs3900-CS3900-GP-Starter)) - not your repo.

To catch this, watch carefully what "main" is referring to.  I'm not going to kill this Pull Request in case you guys have info that needs to be moved to your repo.

Commit:
Updated business needs analysis with new user flow details

StarboardHook committed:
Merge pull request #4 from Ardalan81/update-business-needs

Updated business needs analysis with new user flow details

caraz22 committed:
doing my part of the project requirements document

TheCodeMaster210 reviewed (approved):
Good

caraz22 committed:
adding ardi's questions

caraz22 committed:
Created my interview transcript

caraz22 committed:
made interview markdown

StarboardHook committed:
UPDATED FILE: BR-ProjectRequirementsDocument.md

StarboardHook committed:
UPDATE FILE:
	BR-InterviewQuestions.md - update format/look of the questions, added Ardalan Janpour's questions since we are having access issues with his git and the group repo.

StarboardHook committed:
UPDATED FILE:
	Agile-MPV-CoreFeatures.md - updated format/style, included Ardalan Janpour's portion due to git/repo issues.

hmcintos committed:
Added in the flowchart of our mvp

hmcintos committed:
Update Agile-MVP-CoreFeatures.md

hmcintos committed:
Filled out the scope and requirements sections

hmcintos committed:
fixed a typo

hmcintos committed:
Formatting changes to the scope and Requirements sections

hmcintos committed:
Hunter McIntosh Transcription of interview with Corey Sprouse

AvaMcintosh committed:
Create BR-Interview-Sagiev.md

AvaMcintosh committed:
Update BR-Interview-Sagiev.md

StarboardHook committed:
UPLOAD FILE: BR-Interview-Dauletaliy.md

StarboardHook committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

StarboardHook committed:
UPLOAD FILE: BR-Interview-Dauletaliy.md

AvaMcintosh committed:
Update BR-ProjectRequirementsDocument.md

AvaMcintosh committed:
Update BR-ProjectRequirementsDocument.md

AvaMcintosh committed:
Update BR-ProjectRequirementsDocument.md

AvaMcintosh committed:
Create BR-Interview-Burger.txt

AvaMcintosh committed:
Rename BR-Interview-Burger.txt to BR-Interview-Burger.md

StarboardHook committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter
UPLOAD FILE:
	docker-compose.yml
	init_shelter.sql

StarboardHook committed:
UPDATING FILES

caraz22 committed:
changed line 13

StarboardHook committed:
Update init_shelter.sql

UPDATED:
- line 17 the CHECK constraint was throwing error

StarboardHook committed:
Update init_shelter.sql

caraz22 committed:
updated docker and init files for testing

caraz22 committed:
Merge branch 'main' of github.com:WSU-kduncan/cs3900-animalshelter

caraz22 committed:
changed some syntax/grammar

caraz22 committed:
fixed a typo

caraz22 committed:
i dont know whats wrong but i hope the files are right :(

caraz22 committed:
changed name to pet_name bc name seems to be an sql identifier of some sort

caraz22 committed:
status is also a key word, changed relevant to table

caraz22 committed:
changed date to application_date bc date is an sql keyword

caraz22 committed:
changed datetime to date bc i dont think we need down to the hours minutes and seconds

caraz22 committed:
fixed some formatting

pattonsgirl commented:
@caraz22 @StarboardHook @AvaMcintosh @hmcintos 
Hi guys! You are doing great work, but this Pull Request is pointing to the "golden" repo ([cs3900-f24-cs3900-CS3900-GP-Starter](https://github.com/WSU-kduncan/cs3900-f24-cs3900-CS3900-GP-Starter)) - not your repo.

To catch this, watch carefully what "main" is referring to. I'm not going to kill this Pull Request in case you guys have info that needs to be moved to your repo.

Do not push Contribute from your Animal Shelter repository.  It puts the work back here in the group project starter (template) repository.

hmcintos committed:
fixed adoption application create table statement

caraz22 committed:
Created BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
moved interview to BR folder

caraz22 committed:
Created Agile folder

caraz22 committed:
moved interview to Agile folder

caraz22 committed:
moved interview to Agile folder

caraz22 committed:
Create db-models.md
